docs/contributing/frontend.md

```diff
@@ -237,13 +270,16 @@ another page, you should probably consider using the **E2E** approach.
 
 ### Visual testing
 
-Test components without user interaction like testing if a page view is rendered
-correctly depending on some parameters, if the button is showing a spinner if
-the `loading` props are passing, etc. This should always be your first option
-since it is way easier to maintain. For this, we use
+Test components without user interaction like testing if a page/component is
+rendered correctly depending on some parameters, if the button is showing a
+spinner, if the `loading` props are passing, etc. This should always be your
+first option since it is way easier to maintain. For this, we use
 [Storybook](https://storybook.js.org/) and
 [Chromatic](https://www.chromatic.com/).
 
+> ℹ️ To learn more about testing components that fetch API data, refer to the
```

Comment:
Do we want to use an emoji for this, or would it be better to lean into Github-flavored markdown, once Blueberry starts on that next sprint?

Reply:
I think it is ok for now.